porting/build-sources.rst

```diff
@@ -63,8 +63,17 @@ To check which config options needs to be adjusted we use `mer-kernel-check <htt
 
 If you don't know the path to your kernel config run ``grep "TARGET_KERNEL_CONFIG" device/<VENDOR>/<CODENAME>/BoardConfig.mk``. It should be in ``arch/arm/configs/<CONFIG>`` or ``arch/arm64/configs/<CONFIG>`` depending on the architecture of your device.
 
+.. warning::
+		Make sure that your configuration changes are not overridden by later lines in the config file.
+		
+		Also be aware that ``# CONFIG_IKCONFIG_PROC is not set`` *may* look like a harmless comment, `but it actually unsets`__ ``CONFIG_IKCONFIG_PROC`` if it is set (e.g. by an edit you made earlier in the file).
+		
+		.. __: https://www.tldp.org/HOWTO/SCSI-2.4-HOWTO/kconfig.html
```

Comment:
I prefer not to use 'void' links like this. I'd rather the link text actually described the link's target and the reference was named. For example, the link text could be "See [kernel configuration in the Linux kernel documentation](https://www.tldp.org/HOWTO/SCSI-2.4-HOWTO/kconfig.html)"